src/components/SuggestedAction/SuggestedAction.tsx

```diff
@@ -29,20 +32,30 @@ export interface SuggestedActionProps {
   hideCtas?: boolean;
   /* any children components; optional */
   children?: React.ReactNode;
+  /* these next three props are actions for the hamburger menu, all the arguments are specified in izakaya */
+  /* dismisses the suggestedaction */
+  dismissAction: () => void;
+  /* lets the contractor view the workflow details */
+  viewWorkflow: () => void;
+  /* disables all suggested actions of the specified type */
+  disableAction: () => void;
 }
 
 const SuggestedAction = ({
   variant,
   description,
-  onClickMenu,
   dueDate,
   cta,
   ctaAction,
   secondaryCta,
   secondaryCtaAction,
   hideCtas,
   children,
+  dismissAction,
+  viewWorkflow,
+  disableAction,
```

Comment:
I'm wondering if we could bundle this so there are less disparate props (as it's possible to have different menu button options in the future, I believe)...

but that's also something we can revisit later if needed

Reply:
can do!

Reply:
wait i lied... it won't let me bundle the props

Reply:
ya I wasn't sure if there was a straightforward way to bundle them; it's not really worth the time rn to dig into it so we can just flag for later